roachtest: hibernate failed

roachtest.hibernate [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_Roachtests_RoachtestNightlyIbmBazel/20914601?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_Roachtests_RoachtestNightlyIbmBazel/20914601?buildTab=artifacts#/hibernate) on release-25.4 @ [265cf7ce71b0925349e291bd96f778e2e5aa4664](https://github.com/cockroachdb/cockroach/commits/265cf7ce71b0925349e291bd96f778e2e5aa4664):


```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v25.4.3-dev-265cf7ce71b0925349e291bd96f778e2e5aa4664
Tests run against hibernate 6.6.20
13147 Total Tests Run
13146 tests passed
1 test failed
2495 tests skipped
3 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: org.hibernate.orm.test.jpa.txn.JtaTransactionJoiningTest.testImplicitJoining() - unknown (unexpected)
For a full summary look at the hibernate artifacts 
An updated blocklist (hibernateBlockList) is available in the artifacts' hibernate log
test artifacts and logs in: /artifacts/hibernate/cpu_arch=s390x/run_1
```

Parameters:
 - <code>arch=s390x</code>
 - <code>cloud=ibm</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for ibm clusters_</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58284

--- a/pkg/cmd/roachtest/tests/hibernate_blocklist.go
+++ b/pkg/cmd/roachtest/tests/hibernate_blocklist.go
@@ -22,6 +22,7 @@ var hibernateIgnoreList = blocklist{
 	"org.hibernate.orm.test.batch.BatchTest.testBatchInsertUpdate":                                       "flaky",
 	"org.hibernate.orm.test.bulkid.OracleInlineMutationStrategyIdTest.testDeleteFromEngineer":            "flaky",
 	"org.hibernate.orm.test.hql.BulkManipulationTest.testInsertWithManyToOne":                            "https://hibernate.atlassian.net/browse/HHH-19332",
+	"org.hibernate.orm.test.jpa.txn.JtaTransactionJoiningTest.testImplicitJoining":                       "flaky",
 	"org.hibernate.serialization.SessionFactorySerializationTest.testUnNamedSessionFactorySerialization": "flaky",
 	"org.hibernate.serialization.SessionFactorySerializationTest.testNamedSessionFactorySerialization":   "flaky",
 	"org.hibernate.test.batch.BatchTest.testBatchInsertUpdate":                                           "flaky",

